setup/tue-functions.bash

```diff
@@ -230,29 +230,76 @@ complete -F __tue-git-clean-local _tue-git-clean-local
 #                                              SSH
 # ----------------------------------------------------------------------------------------------------
 
-function _git_https
+function _git_split_https
 {
     local url=$1
-    [[ "$url" == "https://"* ]] && echo "$url" && return 0
+    local web_address=${url#https://}
+    local domain_name=${web_address%%/*}
+    local repo_address=${web_address#*/}
+    repo_address=${repo_address%.git}
+    echo -e "$domain_name\n$repo_address"
+}
+export -f _git_split_https # otherwise not available in sourced files
 
-    local web_address=${url#git@}
+function _git_split_ssh
+{
+    local url=$1
+    local web_address=${url#ssh://}
```

Comment:
A url of this type will fail as domain_name will never be extracted.

Reply:
```
_git_split_ssh ssh://git@github.com:tue-robotics/tue-env
result
github.com
tue-robotics/tue-env
```